<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvGaits</class>
 <widget class="QWidget" name="AdvGaits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>864</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QGridLayout" name="wavelength">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="wavOdd">
       <item>
        <widget class="QLabel" name="wavOddLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Wavelength
(Odd n)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="wavOddSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>71</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="wavOddNeg">
         <property name="text">
          <string>negative</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="wavOddBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.000000000000000</double>
         </property>
         <property name="value">
          <double>8.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="wavCoupled">
       <property name="text">
        <string>Coupled</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="wavEven">
       <item>
        <widget class="QLabel" name="wavEvenLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Wavelength
(Even n)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QSlider" name="wavEvenSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>71</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="wavEvenNeg">
         <property name="text">
          <string>negative</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="wavEvenBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.000000000000000</double>
         </property>
         <property name="value">
          <double>8.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="defaultButton">
     <property name="text">
      <string>Set to Home</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="frequency">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="freqOddGrid">
       <item>
        <widget class="QLabel" name="freqOddLabel">
         <property name="text">
          <string>Frequency
(Odd n)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="freqOddDial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1000</number>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="notchTarget">
          <double>50.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="freqOddNeg">
         <property name="text">
          <string>negative</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="freqOddBox">
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-4.000000000000000</double>
         </property>
         <property name="maximum">
          <double>4.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.005000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="freqEvenGrid">
       <item>
        <widget class="QLabel" name="freqEvenLabel">
         <property name="text">
          <string>Frequency
(Even n)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="freqEvenDial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>4000</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1000</number>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="notchTarget">
          <double>50.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="freqEvenNeg">
         <property name="text">
          <string>negative</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="freqEvenBox">
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-4.000000000000000</double>
         </property>
         <property name="maximum">
          <double>4.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.005000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="freqCoupled">
       <property name="text">
        <string>Coupled</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="phase">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="phaseOddGrid">
       <item>
        <widget class="QLabel" name="phaseOddLabel">
         <property name="text">
          <string>Phase
(Odd n)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="phaseOddDial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="singleStep">
          <number>83</number>
         </property>
         <property name="pageStep">
          <number>125</number>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="notchTarget">
          <double>18.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="phaseOddNeg">
         <property name="text">
          <string>negative</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="phaseOddBox">
         <property name="suffix">
          <string> pi</string>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-2.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="phaseEvenGrid">
       <item>
        <widget class="QLabel" name="phaseEvenLabel">
         <property name="text">
          <string>Phase
(Even n)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="phaseEvenDial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="singleStep">
          <number>83</number>
         </property>
         <property name="pageStep">
          <number>125</number>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="notchTarget">
          <double>20.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QCheckBox" name="phaseEvenNeg">
         <property name="text">
          <string>negative</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="phaseEvenBox">
         <property name="suffix">
          <string> pi</string>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-2.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.005000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="phaseCoupled">
       <property name="text">
        <string>Coupled</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ampLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Amplitude</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTabWidget" name="ampTabModes">
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Sliders</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="ExtendedSliders" name="ampSliders" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1100</width>
           <height>250</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="offsetLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QTabWidget" name="offsetTabModes">
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWidgetPage1" native="true">
      <attribute name="title">
       <string>Sliders</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="ExtendedSliders" name="offsetSliders" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1100</width>
           <height>250</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWidgetPage2" native="true">
      <attribute name="title">
       <string>Master/Slave</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="Replica" name="replicaWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedSliders</class>
   <extends>QWidget</extends>
   <header>qtgui/extendedsliders.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Replica</class>
   <extends>QWidget</extends>
   <header>qtgui/replica.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>defaultButton</tabstop>
  <tabstop>wavOddSlider</tabstop>
  <tabstop>wavOddNeg</tabstop>
  <tabstop>wavOddBox</tabstop>
  <tabstop>wavEvenSlider</tabstop>
  <tabstop>wavEvenNeg</tabstop>
  <tabstop>wavEvenBox</tabstop>
  <tabstop>wavCoupled</tabstop>
  <tabstop>freqOddDial</tabstop>
  <tabstop>freqOddNeg</tabstop>
  <tabstop>freqOddBox</tabstop>
  <tabstop>freqEvenDial</tabstop>
  <tabstop>freqEvenNeg</tabstop>
  <tabstop>freqEvenBox</tabstop>
  <tabstop>freqCoupled</tabstop>
  <tabstop>phaseOddDial</tabstop>
  <tabstop>phaseOddNeg</tabstop>
  <tabstop>phaseOddBox</tabstop>
  <tabstop>phaseEvenDial</tabstop>
  <tabstop>phaseEvenNeg</tabstop>
  <tabstop>phaseEvenBox</tabstop>
  <tabstop>phaseCoupled</tabstop>
  <tabstop>ampTabModes</tabstop>
  <tabstop>offsetTabModes</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>defaultButton</sender>
   <signal>clicked()</signal>
   <receiver>AdvGaits</receiver>
   <slot>resetParams()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetParams()</slot>
  <slot>setMasterOffset()</slot>
 </slots>
</ui>
